fenix/app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -545,6 +547,8 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
                 "finishing" to isFinishing.toString(),
             ),
         )
+
+        PlayStoreAttribution.deferredDeeplinkTime.cancel()
```

Comment:
Isn't this just measuring the amount of time the Activity is active...? How does this help us understand deferred deep link timing?

Reply:
No cancel means we're not measuring that value.  Where `PlayStoreAttribution.deferredDeeplinkTime.stop()` is where we do the measurement.

This cancel call is just nice so if the user leaves our app we cancel the measurement.

Reply:
(Oh, sorry, just noticed this is a cancel)

Reply:
haha sorry again, re-commented before refreshing the page